easybuild/easyconfigs/l/libxc/libxc-3.0.0-foss-2017b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '3.0.0'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+# Results for some functionals (e.g. mgga_c_tpss) deviate with too
+# aggressive optimization settings.
+toolchainopts = {'pic': True, 'lowopt': True}
```

Comment:
Please also use `'optarch': False` and include the accompanying comment, see #5256

Reply:
Don't think optarch false is needed with foss, only intel? according to the intel/2017b version of this.
Which i of course should have based this on in the first place.